### PR TITLE
Us306 fields page

### PR DESCRIPTION
**Pull request by tcyubahi**

### CODE REVIEW

File: app/routes.py [hunk 1]
```diff
@@ -14,6 +14,7 @@
 from app.modules.historicalmodels import *
 from app.modules.constraintmodel import *
 from app.modules.regionmodel import *
+from app.modules.sensorModel import *
```
Comment (kevinjhale) [inline app/routes.py]: Can you instead make this `import app.modules.sensorModel`? You only need to use `from` when there is a specific class you want from the package

File: logs/error.log [hunk 2]
```diff
@@ -0,0 +1,4 @@
+2019-03-18 21:56:03,935 WARNING: 404 Error Resource Not Found [in /Users/tresor/Desktop/SolarSENSE/SolarSENSE/app/routes.py:315]
```
Comment (kevinjhale) [inline logs/error.log]: We don't need your logs. Can you add the logs folder to the `.gitignore` please?

File: app/templates/sensors.html [hunk 3]
```diff
@@ -42,8 +42,30 @@
...
+			    </tr>
+			  </thead>
+			  <tbody>
+			    <tr ng-repeat="sensor in sensors">
+			      <th scope="row">{a sensor.id a}</th>
+			      <td>{a sensor.mac a}</td>
+			      <td>
+			      	<div class="form-group">
+					    <select class="form-control" id="fieldSelect">
+					      <option>1</option>
```
Comment (kevinjhale) [inline app/templates/sensors.html]: This needs to be dynamic from mongo. Let's hold off on approving this PR until that has been integrated

**Comment by kevinjhale** [comment]
app/templates/sensors.html line 40 needs to change from Home to Fields

File: app/static/js/fields.js [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+*/
+
+var app = angular.module('solarsenseApp', []);
+
+app.config(['$interpolateProvider', function($interpolateProvider) {
+  	$interpolateProvider.startSymbol('{a');
+  	$interpolateProvider.endSymbol('a}');
+}]);
+
+app.controller('LinkCtrl', function($scope, $window) {
```
Comment (jortner1) [inline app/static/js/fields.js]: The navigation bar global controller has been pulled into master now, so make sure to remove this controller from this file.

File: app/static/js/sensors.js [hunk 5]
```diff
@@ -0,0 +1,56 @@
+var app = angular.module('solarsenseApp', []);
+
+app.config(['$interpolateProvider', function($interpolateProvider) {
+  	$interpolateProvider.startSymbol('{a');
+  	$interpolateProvider.endSymbol('a}');
+}]);
+
+app.controller('LinkCtrl', function($scope, $window) {
```
Comment (jortner1) [inline app/static/js/sensors.js]: Same thing here, make sure to switch this to use the global navigation bar.

File: app/templates/fields.html [hunk 6]
```diff
@@ -0,0 +1,75 @@
...
+	<body ng-controller="FieldsCtrl">
+		<div class="loadingOverlay" ng-show="showLoader">
+			<div class="loader"></div>
+		</div>
+		<div class="header">
+			<div class="container">
+				<div class="logo pull-left"><h4 class="vertical" style="margin-left: 44px;color: #FFF;">SolarSENSE</h4></div>
+			</div>
+			<div class="clearfix"></div>
+			<div class="container">
```
Comment (jortner1) [inline app/templates/fields.html]: Update to use Global Nav Bar.

File: logs/error.log [hunk 7]
```diff
@@ -0,0 +1,4 @@
+2019-03-18 22:16:09,931 WARNING: 404 Error Resource Not Found [in /Users/tresor/Desktop/SolarSENSE/SolarSENSE/app/routes.py:314]
```
Comment (jortner1) [inline logs/error.log]: Make sure to remove these error logs.

File: app/templates/fields.html (hunk 6 above)
Comment (wjdavis1) [reply app/templates/fields.html]: Why is this Global nav bar a route to the flask server, why not just create it as a base template and have that included in the different html forms

File: app/static/js/sensors.js (hunk 5 above)
Comment (wjdavis1) [reply app/static/js/sensors.js]: This has been resolved with merging the change in dev into this branch

File: app/static/js/fields.js (hunk 4 above)
Comment (wjdavis1) [reply app/static/js/fields.js]: This has been resolved with merging the change in dev into this branch

File: logs/error.log (hunk 7 above)
Comment (wjdavis1) [reply logs/error.log]: These have been removed and added to the gitignore file

File: app/templates/sensors.html (hunk 3 above)
Comment (wjdavis1) [reply app/templates/sensors.html]: I will be working this in my fixes to the Settings Page PR so stay tuned

**Pull request merged**